automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestUnits.scala

```diff
@@ -35,7 +33,7 @@ class TestUnits extends ExtractionTest {
   }
 
   val t4a = "In one type, water uptake is a function of the difference in water potentials ( , J kg−1) and the conductances (C, kg s m−4) between adjacent components in the soil–plant system."
-  passingTest should s"extract variables and units from t4a: ${t4a}" taggedAs(Somebody) in {
+  failingTest should s"extract variables and units from t4a: ${t4a}" taggedAs(Somebody) in {
```

Comment:
because of concept expansion - after this is merged, we could try playing with allowed incoming/outgoing relations. I don't want to do it now because this impacts functions and you may have already adjusted this in a different branch.

Reply:
Sounds good. Let's try adjusting valid incoming/outgoing relations once this branch is merged.